packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1432,10 +1432,10 @@ class DropdownButtonFormField<T> extends FormField<T> {
            );
            return InputDecorator(
              decoration: effectiveDecoration.copyWith(errorText: field.errorText),
-             isEmpty: value == null,
+             isEmpty: field.value == null,
```

Comment:
Could you also add a test to verify that this behaves as expected?

Reply:
I don't know how to access the `InputDecorator`'s properties from the tests, could you give a pointer please?

Reply:
The first test in `input_decorator_test.dart` has a section where it checks the `InputDecorator`'s properties when `isEmpty` is true.